src/lib/theme/default.ts

```diff
@@ -138,4 +138,49 @@ export default {
       },
     },
   },
+  spinner: {
+    base: 'inline animate-spin text-gray-200',
+    color: {
+      failure: 'fill-red-600',
+      gray: 'fill-gray-600',
+      info: 'fill-blue-600',
+      pink: 'fill-pink-600',
+      purple: 'fill-purple-600',
+      success: 'fill-green-500',
+      warning: 'fill-yellow-400',
+    },
+    light: {
+      off: {
+        base: 'dark:text-gray-600',
+        color: {
+          failure: '',
+          gray: 'dark:fill-gray-300',
+          info: '',
```

Comment:
why do we have all this blank properties?

Reply:
The type for `spinner.color` is reused for the `color` of other items inside the spinner, so it's expecting all of the same base colors in each sub type (in this case, `light`) as well. I can add interfaces for the subtypes, I'm not really sure either way which makes more sense.

Reply:
OK. I was just curious. I don't see it as a problem.